src/components/screens/ScoreSetCreator.vue

```diff
@@ -95,15 +95,25 @@
                       </div>
                       <div v-else style="position: relative;">
                         <span class="p-float-label">
-                          <Dropdown
+                          <AutoComplete
+                            ref="experimentInput"
                             v-model="experiment"
+                            dropdown
                             :id="$scopedId('input-experiment')"
+                            :suggestions="experimentSuggestionsList"
+                            field="title"
+                            forceSelection
+                            :multiple="false"
                             :options="editableExperiments"
```

Comment:
I think another nice addition here would be to use the virtual scroller:
```
:virtualScrollerOptions="{ itemSize: 50 }"
```

Although it might not be super useful for me and you on staging, it will be nice for the users who have a huge amount of experiments since it will make them load much quicker.